nepalitime/nepalitime.go

```diff
@@ -32,60 +35,299 @@ func (obj NepaliTime) String() string {
 }
 
 // Get's the corresponding english time
-func (obj NepaliTime) GetEnglishTime() *time.Time {
+func (obj *NepaliTime) GetEnglishTime() *time.Time {
```

Comment:
Adding pointer on the `NepaliTime` object might lead to the bad state of the `englishTime`. Let's not use it here.